crest/Assets/Crest/Crest/Scripts/LodData/Settings/SimSettingsAnimatedWaves.cs

```diff
@@ -27,6 +27,11 @@ public partial class SimSettingsAnimatedWaves : SimSettingsBase
         float _attenuationInShallows = 0.95f;
         public float AttenuationInShallows => _attenuationInShallows;
 
+        [Tooltip("Any deeper than this value will have no attenuation (ie full wave spectrum). Set to maximum value (1,000) to disable. If it is used, it will reduce the overall attenuation effectiveness.")]
+        [SerializeField, Range(0f, LodDataMgrSeaFloorDepth.k_DepthBaseline)]
```

Comment:
it may be we should not allow this to be 0 to avoid div by 0, (or clamp it before giving it to the shader)

Reply:
I made minimum 1. I doubt anyone wants anything below that and nice and round.